src/components/Navigation/TopBar/ApplicationDrawer/ApplicationDrawer.test.tsx

```diff
@@ -96,6 +100,26 @@ test('Should toggle menu and handle application click', async () => {
   }
 });
 
+test('background color is shown for the app you are in', async () => {
+  rejectPromise = false;
+  vi.stubEnv('VITE_NAME', fakeApps[0].name);
+  render(<ApplicationDrawer />, { wrapper: Wrappers });
+
+  const user = userEvent.setup();
+
+  const menuButton = await screen.getByRole('button');
+
+  await user.click(menuButton);
+
+  await waitForElementToBeRemoved(() => screen.getByRole('progressbar'), {
+    timeout: 4000,
+  });
+  const firstAppContainer = screen.getByText(fakeApps[0].name);
+  expect(firstAppContainer).toHaveStyleRule(
+    'background-color: colors.interactive.primary__selected_highlight.hex'
```

Comment:
Does this work? 😮 I though you needed to do this: `expecte(elemen).toHaveStyleRule('background', colors....)`?